README.md

```diff
@@ -1,60 +1,76 @@
 # Trapperkeeper-authorization
 
-[![Build Status](https://travis-ci.org/masterzen/trapperkeeper-authorization.svg?branch=master)](https://travis-ci.org/masterzen/trapperkeeper-authorization)
+[![Build Status](https://travis-ci.org/masterzen/trapperkeeper-authorization.svg?branch=master)](https://travis-ci.org/puppetlabs/trapperkeeper-authorization)
```

Comment:
Should the first URL here also be `https://travis-ci.org/puppetlabs/trapperkeeper-authorization...`?